tests/reconfig/results/v1.0.0.md

```diff
@@ -0,0 +1,42 @@
+# Reconfiguration testing Results
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.4-gke.900
+- Zone: europe-west2-b
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+NGF deployment:
+
+- NGF version: edge - git commit 72b6c6ef8915c697626eeab88fdb6a3ce15b8da0
+- NGINX Version: 1.25.2
+
+## Results Table
+
+| Test number | NumResources | TimeToReadyTotal | TimeToReadyAvgSingle | NGINX reloads (total) | NGINX reload avg time (ms) |
```

Comment:
Can you add the units for the time to ready columns?
Also, for simplicity, I think it's ok to include the initial reload in the NGINX reloads total column.